polar-core/benches/bench.rs

```diff
@@ -169,91 +167,10 @@ pub fn many_rules(c: &mut Criterion) {
     });
 }
 
```

Comment:
I have no idea how bench tests work so not reviewing these changes.

Reply:
They were a mess and overly complicated. So I moved the one that benched externals to be a bench in the oso library instead, so that its way simpler.

Reply:
In case you're interested, this is how it looks now:
https://github.com/osohq/oso/pull/515/files#diff-ccf3c4f853abe6996692d38527d164a48366a51ed18810306292370442de2abdR73

Instead of all that fake external stuff, we can just create a new struct with whatever implementation we want